deploy/fb_dev.rules

```diff
@@ -0,0 +1,10 @@
+service cloud.firestore {
```

Comment:
If I remember correctly this thing needs to have a specific name and in the same folder as the other fb config setup code?

Reply:
Ahh i didnt realise the position of this file in the fs is important in the deployment. I was under the impression the deploy sh can copy this as require - will update this.

Reply:
I could well be wrong, worth checking in the firebase docs. I was only replying from memory...